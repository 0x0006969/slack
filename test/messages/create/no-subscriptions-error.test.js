const NoSubscriptionsError = require('../../../lib/messages/create/no-subscriptions-error');

describe('NoSubscriptionsError message rendering', () => {
  test('works', async () => {
    const message = new NoSubscriptionsError();
    expect(message.getAttachment()).toMatchSnapshot();
  });
});
