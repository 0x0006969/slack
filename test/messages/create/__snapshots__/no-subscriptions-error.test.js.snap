// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoSubscriptionsError message rendering works 1`] = `
Object {
  "attachments": Array [
    Object {
      "color": "danger",
      "mrkdwn_in": Array [
        "text",
      ],
      "text": "This channel is not subscribed to any repositories",
    },
  ],
  "response_type": "ephemeral",
}
`;
